tests/e2e/test.sh

```diff
@@ -41,14 +43,21 @@ function local_setup {
     trap local_cleanup EXIT
     selenium_start
     sleep 5
+}
 
-    add_test_users
+function reset_db {
+    cd ../mattermost-server
+
+    echo "reset the database"
+    go run $PLATFORM_FILES reset --confirm true
```

Comment:
Is this intended to be something that devs run before submitting a PR, or is it just going to be run on a dev server? I wouldn't want to run this if it reset my database each time

Reply:
Goal is to have this part of our PR build.  At the moment, not a requirement to run before submitting a PR.
That's a good point. I'll submit separate PR to address that before we integrate E2E in our build system.